spec/models/user_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+describe User do
+  
+  describe '#card_for_review' do
+    let(:user) { create(:user) }
+    let(:block1) { create(:block, user: user) }
+    let(:block2) { create(:block, user: user) }
+    let!(:card1) { create(:card, user: user, block: block1) }
+    let!(:card2) { create(:card, user: user, block: block2) }
+
+    context 'when user has current block' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.